flexinput/src/main/java/com/lytefast/flexinput/fragment/CameraFragment.java

```diff
@@ -90,29 +100,19 @@ void onTakePhotoClick() {
   void onLaunchCameraClick() {
     cameraView.stop();
 
-    final File photoFile = fileManager.newImageFile();
-    photoTakenCallback.onPhotoTaken(photoFile);
+    final File photoFile = flexInputCoordinator.getFileManager().newImageFile();
+    flexInputCoordinator.onPhotoTaken(FileUtils.toAttachment(photoFile));
 
     Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE)
-        .putExtra(MediaStore.EXTRA_OUTPUT, fileManager.toFileProviderUri(getContext(), photoFile))
+        .putExtra(MediaStore.EXTRA_OUTPUT,
+                  flexInputCoordinator.getFileManager().toFileProviderUri(getContext(), photoFile))
```

Comment:
You'll probably need to do the same hack here for older devices to be able to write to the URI.